lib/services/appmanager-service.ts

```diff
@@ -99,5 +139,31 @@ class AppManagerService implements IAppManagerService {
 			}
 		}).future<void>()();
 	}
+
+	private findGroups(identityStrings:string[]): IFuture<string[]> {
+		return ((): any => {
+			let availableGroups = this.$server.tam.getGroups().wait();
+			let groupIds: string[] = [];
+
+			if (!availableGroups.length) {
+				this.$errors.failWithoutHelp("No available groups found.");
+			}
+
+			_.forEach(identityStrings, (identityStr, index) => {
```

Comment:
I've not tested the code above, so if you decide to use it, please test it :smile_cat: